SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -277,10 +288,9 @@ void USpatialNetDriver::OnAcceptingPlayersChanged(bool bAcceptingPlayers)
 			// Extract map name and options
 			FWorldContext* WorldContext = GEngine->GetWorldContextFromWorld(GetWorld());
 			check(WorldContext);
-			
-			FURL RedirectURL = FURL(&WorldContext->LastURL, *GlobalStateManager->DeploymentMapURL, (ETravelType)WorldContext->TravelType);
-			RedirectURL.Host = WorldContext->LastURL.Host;
-			RedirectURL.Port = WorldContext->LastURL.Port;
+
+			FURL RedirectURL = WorldContext->LastURL;
+			RedirectURL.Map = GlobalStateManager->DeploymentMapURL;
```

Comment:
If the MapURL is passed in via the constructor, FURL does a ton of processing on it that we lose by assigning direct to Map.